Sources/Tests/.clang-tidy

```diff
@@ -0,0 +1,2 @@
+InheritParentConfig: true
```

Comment:
```
[89/100] Running clang-tidy on test_CoreLib...
YAML:1:22: error: unknown key 'InheritParentConfig'
InheritParentConfig: true
                     ^~~~
```

Reply:
It looks like this option is [relatively new](https://reviews.llvm.org/D75184).

Reply:
That would explain it...  I'm using whatever version ships with MSVC

Reply:
The best I can do is disable sanitizing the tests, unless you can write a regex to filter out `Sources/Tests/gtests` but not `Sources/Plasma` and `Sources/Tools`...